reps_event_app/lib/api/reps_rep_api.dart

```diff
@@ -1,6 +1,18 @@
-
+import 'dart:convert';
+import 'package:http/http.dart' as http;
 
 Future<dynamic> fetchReps({String date,String cityName}) async {
```

Comment:
@devanshgajjar please resolve this issue.

Reply:
What do I have to do here?

Reply:
We do not need `date` and `cityName` in the `fetchReps` method, this was meant for events page only. If you are taking reference of some other code snippet or file, make sure you don't include unnecessary code.